llama_index/async_utils.py

```diff
@@ -1,13 +1,39 @@
 """Async utils."""
 import asyncio
 from typing import Any, Coroutine, List
+import nest_asyncio
```

Comment:
If we decide on importing this, should add dependency to `setup.py`

Reply:
makes sense. as per @jerryjliu's note below though I'll just remove `nest_asyncio`